README.md

```diff
@@ -1,8 +1,15 @@
 # OpenFGA
 
-[![Go Reference](https://pkg.go.dev/badge/github.com/openfga/openfga.svg)](https://pkg.go.dev/github.com/openfga/openfga) ![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/openfga/openfga?sort=semver&color=green) [![Container Image](https://img.shields.io/github/v/release/openfga/openfga?color=blueviolet&label=container&logo=docker "Container Image")](https://hub.docker.com/r/openfga/openfga/tags) ![Downloads](https://img.shields.io/github/downloads/openfga/openfga/total.svg?style=flat&color=lightgrey) [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](./LICENSE)
-![Codecov](https://img.shields.io/codecov/c/github/openfga/openfga) ![Snyk Vulnerabilities for GitHub Repo](https://img.shields.io/snyk/vulnerabilities/github/openfga/openfga?color=orange)
-[![Discord Server](https://img.shields.io/discord/844600078504951838?color=7289da&logo=discord "Discord Server")](https://discord.com/channels/759188666072825867/930524706854031421) [![Twitter](https://img.shields.io/twitter/follow/openfga?color=%23179CF0&logo=twitter&style=flat-square "@openfga on Twitter")](https://twitter.com/openfga)
+[![Go Reference](https://pkg.go.dev/badge/github.com/openfga/openfga.svg)](https://pkg.go.dev/github.com/openfga/openfga)
```

Comment:
I put each badge on its own line. Easier to read the markdown. Still displays on one line.

Reply:
This was intentional though, because now all of the badges are on newlines in the README
<img width="458" alt="Screen Shot 2022-06-16 at 1 39 08 PM" src="https://user-images.githubusercontent.com/2899204/174159646-b3b5f41c-b59b-4b1d-a87d-8c94348fcc63.png">

They were purposefully categorized on lines

<artifacts>
<repo health>
<social>

Reply:
Fixed!